src/libs/actions/IOU.ts

```diff
@@ -5116,6 +5200,11 @@ function requestMoney(requestMoneyInformation: RequestMoneyInformation) {
             break;
         }
         default: {
+            const guidedSetupData: GuidedSetupData | undefined =
+                isTestDrive && introSelected?.choice
```

Comment:
I'm thinking that `introSelected.choice` here isn't relevant? If we end up here with `isTestDrive` then the choice that we are passing is gonna be hardcoded. Since the setup data we need is for the boss we are inviting.

It should always just be the`ONBOARDING_MESSAGES[onboardingChoices.TEST_DRIVE_RECEIVER]`.